src/utils/I18nUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { Alert, NativeModules } from 'react-native';
+import RNRestart from 'react-native-restart';
+import rtlDetect from 'rtl-detect';
+
+export const languageRestart = async (prevLang, lang) => {
+  if (prevLang != lang) {
+    // if selected lang is RTL, switch the layout
+    if (rtlDetect.isRtlLang(lang)) {
+      await NativeModules.I18nManager.forceRTL(true);
+    } else {
+      await NativeModules.I18nManager.forceRTL(false);
+    }
+
+    Alert.alert(
+      'Restart Ecency?',
```

Comment:
@aliseyalvi strings should always be added in `en-US.json` file and referenced using `useInl()`.. for example implementation you can check `commentsView.tsx`

Reply:
updated. https://github.com/ecency/ecency-mobile/pull/2116/commits/79539637755f2133239ab47f43ff49fcb3b7e6ea